frontend/packages/dev-console/integration-tests/features/build-config/edit-build-config.feature

```diff
@@ -0,0 +1,94 @@
+
+
+        @smoke @to-do
+        Scenario: Edit Build Config page: EBC-01-TC01
+            Given user is at Topology page
+             When user navigates to build tab
+              And user clicks on kebab menu for "nodejs-ex-git1" build config
+              And user clicks on Edit BuildConfig
+             Then user will see the Git Repository url, Image configuration, Environment Variables
+
+
+        @regression @to-do
+        Scenario: Advanced options in Edit Build Config page: EBC-01-TC02
+            Given user is at Build tab
+             When user clicks on kebab menu for "nodejs-ex-git1" build config
+              And user clicks on Edit BuildConfig
+              And user clicks on Show advanced options
+             Then user will see the Triggers, Build Secret, Run Policy, Post-Commit Hooks
+
+
+        @regression @manual
+        Scenario Outline: Switch from Form to YAML view for editing BuildConfig: EBC-01-TC03
+            Given user is at Build tab
+             When user clicks on kebab menu for "nodejs-ex-git1" build config
+              And user clicks on Edit BuildConfig
+              And user switches to YAML view
+              And user changes spec.output.to.name to "nodejs-ex-git:2.0"
+              And user switches to Form view
+             Then user will see tag in Push to value as "nodejs-ex-git:2.0"
+
+
+        @regression @to-do
+        Scenario Outline: Edit a buildconfig which uses strategy Source(S2I)) and Git as source: EBC-01-TC04
+            Given user is at Edit Build Config page of deployment "nodejs-ex-git1"
+             When user changes Git Rpository URL to "<git_url>" menu of build config
+              And user selects imagestream "<image_stream>" and tag "<tag>" in Build From section in Image Configuration
+              And user enters Name and Value as "<name>" and "<value>" respectively in Environment Variables
+              And user clicks Save button on Edit build Config page
+             Then user will see Git Repository as "<git_url>" and Build from as "<image_stream>:<tag>"
+
+        Examples:
+                  | git_url                                 | image_stream | tag | name | value |
+                  | https://github.com/sclorg/django-ex.git | python       | 3.8 | path | /home |
+
+
+        @regression @manual
+        Scenario: Edit Advanced git options of buildconfig which uses strategy Docker and Git as source: EBC-01-TC05
+        # user can use buildconfig-with-strategy-docker-source-git.yaml from testData/yamls/BuildConfig
+            Given user has applied the YAML buildconfig-with-strategy-docker-source-git.yaml
+              And user is at Edit Build Config page
+             When user clicks Show advanced Git options
+              And user changes value of Context Dir to "beginner/static-site"
+              And user clicks Save button on Edit Build Config page
+             Then user will see Context dir as "beginner/static-site"
```

Comment:
Can't we automate this? It seems to me that we can automate all the statements, the first one as well. Same for the following scenario as well.

Reply:
I thought we are not doing this in Add flows so marked as manual. Updated